CHANGELOG.md

```diff
@@ -23,6 +23,8 @@ changes.
 
 - **BREAKING** The `hydra-cluster` executable (our smoke test) does require `--publish-scripts` or `--hydra-scripts-tx-id` now as it may be provided with pre-published hydra scripts.
 
+- **BREAKING** The `InitialSnapshot` only contains the `initialUTxO` as the rest of the information was useless.
```

Comment:
I'd appreciate a review here. Do I give enough information with this change log?

/cc @ch1bo